doc/usage/restructuredtext/basics.rst

```diff
@@ -293,6 +302,7 @@ Docutils supports the following directives:
   - :dudir:`highlights`, :dudir:`pull-quote` (block quotes with their own
     class attribute)
   - :dudir:`compound <compound-paragraph>` (a compound paragraph)
+  - :dudir:`code-block` (literal blocks with a specific syntax)
```

Comment:
No, this works now since at least 0.14. See [here](http://repo.or.cz/docutils.git/blob/refs/tags/docutils-0.14:/docutils/parsers/rst/languages/en.py#l23). Did I miss something?

Reply:
Ok, so it's just an alias. My mistake. Given that we don't support the 'code' directive, as you note, I'll just drop this.

Reply:
Done.